test/unit/resolve-rewrites.test.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-env jest */
+import resolveRewrites from 'next/dist/shared/lib/router/utils/resolve-rewrites'
+import { removeTrailingSlash } from 'next/dist/shared/lib/router/utils/remove-trailing-slash'
+
```

Comment:
If we go with the solution of removing `strict: true` from path-to-regex then we might not want to bother with this test file as it is really just testing that `strict: false` of path-to-regex is working correctly.